reascripts/ReaSpeech/source/ReaSpeechMain.lua

```diff
@@ -16,38 +16,25 @@ function ReaSpeechMain:main()
   ctx = ImGui.CreateContext(ReaSpeechUI.TITLE, ReaSpeechUI.config_flags())
   Fonts:load()
   app = ReaSpeechUI.new()
+  app:open()
+
   reaper.defer(self:loop())
 end
 
 function ReaSpeechMain:loop()
-  local visible, open = false, false
-
   return function()
     ImGui.PushFont(ctx, Fonts.main)
-    app:trap(function()
```

Comment:
I think we still want an app:trap here, so that `PopFont` gets called reliably.

Reply:
moving this font push/pop to another wrapping helper like with the theme